### DISCUSSION TITLE
Please test this project on your end and get back to me asap :)

### DISCUSSION THREAD

**Discussion by Karem1986** [EddieHubCommunity/support#158] [Q&A] [answered]
https://polshop.netlify.app/ 

Please test that when you click you are able to see all products being loaded on the page and that everything works like pages and add to shopping cart :))

Please let me know once you finish

**Comment by eddiejaoude** [comment]
Yes it works fine 👍 

The step I took:

1. Open the link
2. List of products appear
3. Add to basket
4. Click on basket
5. Items I selected are there

**Reply by Karem1986** [reply]
Thanks so much Eddie, this is the project I am going to show to employers and recruiters :)

**Comment by AakarshTeja** [comment]
![image](https://user-images.githubusercontent.com/42816874/92554751-b0dafa80-f283-11ea-988a-0cdb294ba280.png)

When I tried to submit a review..404 page is showing up and when I click that back to site...its not redirecting

**Comment by amendezm** [comment, marked as answer]
<img width="960" alt="image" src="https://user-images.githubusercontent.com/26444448/179580778-180519c0-ab4d-4e53-b321-2bab86f9d8cd.png">

**Reply by kohei0802** [reply]
Same Result.

**Reply by Karem1986** [reply]
Database issue due to not being pro/paid. Thanks!

**Comment by dinxsh** [comment]
# Pages that work 

the website loaded properly after an error

the registration/login page doesnt work

# Pages that dont work 

the cart page show's a hello world boiler text which mean it's working properly

the home page showed an error with heading

# Errors 

around 10+ errors on console while fetching - mostly errors in fetch, failed to load resource

# Suggestions

> solve all console errors
> fix register/login page (solve possible db errors)
> make logo transparent
> add content on home page

if my answer was helpful kindly mark it as answer :)

**Comment by lucky18210** [comment]
It's okay

**Comment by LR-TechX** [comment]
Only the error page appeared after multiple trial... Check your condition statement🤔🤝🏻

**Comment by Karem1986** [comment]
Thanks a lot everyone who reviewed this project! 
The error is due to the data base not working on Heroku, they changed their plans to paid so  I decided to not pay for the maintenance of the data base on it.
I will close now this thread, but upvote all answers. If I can help with Java projects on Github feel free to get in touch :)

**Reply by LR-TechX** [reply]
Nice, keep up the good work🥂🤩
